src/client/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -158,6 +158,11 @@ public string LoginHint
         public string LongRunningOboCacheKey { get; set; }
 
         public KeyValuePair<string, string>? CcsRoutingHint { get; set; }
+
+        /// <summary>
+        /// Represents the client info of the account returned from the authorization enpoint.
```

Comment:
I was thinking about this and I am wondering if it is better to validate against the client info recieved from the authorize endpoint instead of removing the check entirely. Seems like this approach can provide some benefit. 
What are your thoughts @bgavrilMS? It does work with manual testing. Are there any situations where the client info returned from the authorize endpoint will be different from the client info returned from the token endpoint?

Reply:
What is the benefit of this verification? How does it provide more value than PKCE?